### DISCUSSION TITLE
Do not break after" list for German?

### DISCUSSION THREAD

**Discussion by HeikeMuc** [SubtitleEdit/subtitleedit#7683] [Q&A] [answered]
Hi guys,
Is there a way to create a "Do not break after" list for German? Or can I tell SE that my text is English so that I put my words into the English list? 
Thanks for help.

**Comment by niksedk** [comment, marked as answer]
Go to your SE data folder (press Ctrl+shift+alt+D) and navigate to the "Directories" folder.

Here make a `de_NoBreakAfterList.xml` file (take a copy of one of the other files):
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/9f0b9bf9-74e7-418b-9f3c-fb0ce8c3f9cd)

Could you make a short list of the most obvious German do-not-break-after words?

**Reply by HeikeMuc** [reply]
Oh, thanks, that's great! Here is the list which has to be continued:

<NoBreakAfterList>
  <Item>Absatz</Item>
  <Item>Artikel</Item>
  <Item>Dr.</Item>
  <Item>Ihr</Item>
  <Item>Mr.</Item>
  <Item>Mrs.</Item>
  <Item>Ms.</Item>
  <Item>Nr.</Item>
  <Item>Sie</Item>
  <Item>aber</Item>
  <Item>als</Item>
  <Item>auch</Item>
  <Item>bis</Item>
  <Item>das</Item>
  <Item>dass</Item>
  <Item>dein</Item>
  <Item>deine</Item>
  <Item>deinem</Item>
  <Item>deinen</Item>
  <Item>deiner</Item>
  <Item>deines</Item>
  <Item>der</Item>
  <Item>die</Item>
  <Item>du</Item>
  <Item>ein</Item>
  <Item>eine</Item>
  <Item>einem</Item>
  <Item>einen</Item>
  <Item>einer</Item>
  <Item>eines</Item>
  <Item>gar</Item>
  <Item>im</Item>
  <Item>in</Item>
  <Item>mit</Item>
  <Item>ob</Item>
  <Item>obwohl</Item>
  <Item>oder</Item>
  <Item>so</Item>
  <Item>um</Item>
  <Item>und</Item>
  <Item>von</Item>
  <Item>weil</Item>
  <Item>wie</Item>
  <Item>zu</Item>
  <Item>§</Item>
  <Item>Abs.</Item>
  <Item>Art.</Item>
  <Item>Nummer</Item>
  <Item>solange</Item>
  <Item>soweit</Item>
</NoBreakAfterList>

**Comment by HeikeMuc** [comment]
I wonder if it would be possible to write a regular expression into the list. Many people have the problem that a line ends with a comma or a fullstop, and behind it one single word. So one could say "Fullstop, space, and behind it at least 1 or 2 letters, maximum 10 letters" or so. I am not familiar with regex but maybe someone knows.

**Comment by niksedk** [comment]
SE has a few more settings for line break:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/601e50f2-2352-4d87-af11-b1da5ef04874)

German no-break-after-list added here: https://github.com/SubtitleEdit/subtitleedit/commit/0a7642cd9897497e830f84f3057a95d09d648d22
